### PR TITLE
Docs changes for launch.py usage

### PR DESCRIPTION
**Pull request by anirudh2290**
Additional information on launch.py usage @eric-haibin-lin @piiswrong @madjam

### CODE REVIEW

File: docs/FAQ/multi_devices.md [hunk 1]
```diff
@@ -148,6 +148,13 @@ Note that here we
 - `-n` number of worker nodes to run on
 - `-H` the host file which is required by `ssh` and `mpi`
 - `--kv-store` use either `dist_sync` or `dist_async`
+- If the `-s` argument is not passed, it will keep the number of servers same as number of workers
```
Comment (eric-haibin-lin) [inline docs/FAQ/multi_devices.md]: Should we first mention what `-s` is for?

File: docs/FAQ/multi_devices.md [hunk 2]
```diff
@@ -148,6 +148,13 @@ Note that here we
...
+- The launch.py script tries to cycle through the hosts file to launch the servers and workers. For eg.
+  Lets say you have `5` hosts in the hosts file and you passed n as `3`(and nothing for s).
```
Comment (eric-haibin-lin) [inline docs/FAQ/multi_devices.md]: `For eg. Lets say ` -> `For example, let's say`

File: docs/FAQ/multi_devices.md (hunk 1 above)
Comment (anirudh2290) [reply docs/FAQ/multi_devices.md]: Added!

File: docs/FAQ/multi_devices.md (hunk 2 above)
Comment (anirudh2290) [reply docs/FAQ/multi_devices.md]: Updated.

**Comment by eric-haibin-lin** [comment]
Could you sync w/ master?

File: docs/faq/multi_devices.md [hunk 3]
```diff
@@ -148,6 +148,14 @@ Note that here we
 - `-n` number of worker nodes to run on
 - `-H` the host file which is required by `ssh` and `mpi`
 - `--kv-store` use either `dist_sync` or `dist_async`
+- `-s` number of server nodes to run on
+- If the `-s` argument is not passed, it will keep the number of servers same as number of workers
+- The launch.py script tries to cycle through the hosts file to launch the servers and workers. For example,
+  let's say you have `5` hosts in the hosts file and you passed n as `3`(and nothing for s).
+  The script will launch `3` server processes for the first three hosts in the hosts file and launch `3` worker
```
Comment (piiswrong) [inline docs/faq/multi_devices.md]: This sounds like you are launching 3 processes for each of the first three hosts

Comment (anirudh2290) [reply docs/faq/multi_devices.md]: thanks for pointing! i have changed this.

**Comment by anirudh2290** [comment]
@piiswrong @eric-haibin-lin  is this good to merge ?

File: docs/faq/multi_devices.md [hunk 4]
```diff
@@ -148,6 +148,14 @@ Note that here we
...
 - `-H` the host file which is required by `ssh` and `mpi`
 - `--kv-store` use either `dist_sync` or `dist_async`
+- `-s` number of server nodes to run on
+- If the `-s` argument is not passed, it will keep the number of servers same as number of workers
+- The launch.py script tries to cycle through the hosts file to launch the servers and workers. For example,
+  let's say you have `5` hosts in the hosts file and you passed n as `3`(and nothing for s).
+  The script will launch a total of `3` server processes, one each for the first three hosts and
+  launch a total of `3` worker processes, one each for the fourth, fifth and first host.
+- If the hosts file has exactly `n` number of worker nodes which is passed as an argument with `-n`, it will launch
+  a server and worker process on each of the `n` hosts.
```
Comment (eric-haibin-lin) [inline docs/faq/multi_devices.md]: a server process and a worker process

**Comment by anirudh2290** [comment]
@eric-haibin-lin Done!

**Pull request merged**